src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -1574,3 +1574,135 @@ fixCursorForBbfTableData(Archive *fout,
 	*nfields_new = fixCursorForBbfSqlvariantTableData(fout, tbinfo, buf, *nfields,
 													  sqlvar_metadata_pos);
 }
+
+/*
+ * babelfishDumpOpclassHelper - This particular helper function would be helpful
+ * to dump Babelfish operator class created on the top of built-in data types of
+ * the Postgres. For example, Operator class sys.int_numeric for the type int4.
+ * This is needed because of some bug in postgres due to which Operarors and
+ * Support functions are not being dumped correctly for user defined operator
+ * class defined for built-in datatypes.
+ */
+void
+babelfishDumpOpclassHelper(Archive *fout, const OpclassInfo *opcinfo, PQExpBuffer *buff, bool *needComma)
```

Comment:
`PQExpBuffer` is itself a pointer so we don't need to pass the address of `buff`. Can simply use `PQExpBuffer buff` here.

Reply:
Addressed in new revision